lxd/instance/drivers/driver_qemu.go

```diff
@@ -2377,6 +2379,8 @@ func (d *qemu) addNetDevConfig(sb *strings.Builder, bus *qemuBus, bootIndexes ma
 	} else if shared.PathExists(fmt.Sprintf("/sys/class/net/%s/tun_flags", nicName)) {
 		// Detect TAP (via TUN driver) device.
 		tplFields["ifName"] = nicName
+		tplFields["vectors"] = 2*cpuCount + 2
```

Comment:
@stgraber could you add a comment here explaining the calculation?

Reply:
I would if I understood it ;)

It's number of CPU * 2 for RX/TX presumably, but no idea where the +2 comes from other than it being something you should add ;)

Reply:
Haha, ok fair enough. What is the difference between a vector and a queue?

Reply:
Added some comments now.

Reply:
@stgraber this may be useful to add as a comment:

https://www.linux-kvm.org/page/Multiqueue

"enable mq and specify msix vectors in qemu cmdline (2N+2 vectors, N for tx queues, N for rx queues, 1 for config, and one for possible control vq)"

Reply:
updated again with that :)